pkg/server/cloud/errors.go

```diff
@@ -24,6 +24,7 @@ type NoCapacityError struct {
 	OriginalError string
 	AZ            string
 	SubnetID      string
+	InstanceType  string
```

Comment:
Is this left over? This field does not seem to be referenced anywhere in this PR.

Reply:
nice catch :+1: